xml/nm.xml

```diff
@@ -33,6 +33,17 @@ http://www.heise.de/netze/artikel/WLAN-und-LAN-sichern-mit-IEEE-802-1X-und-Radiu
   active at once. This enables you to unplug your laptop from an Ethernet and
   remain connected via a wireless connection.
  </para>
+
+<important>
+ <title></title>
+ <para>
+ &nm; is only supported by &suse; for desktop workloads with &sleda; or
+ Workstation extension. All server certifications are done with wicked as the
```

Comment:
Again, <command>wicked</command> or similar formatting.

I suggest this as it is both an adjective in the position/role of a noun, and also because in very colloquial English "wicked" is used as a term of praise meaning "very good", thus this usage position of the word could cause confusion.